internal/ir/xds.go

```diff
@@ -1375,6 +1375,8 @@ type DestinationEndpoint struct {
 	Port uint32 `json:"port" yaml:"port"`
 	// Path refers to the Unix Domain Socket
 	Path *string `json:"path,omitempty" yaml:"path,omitempty"`
+	// Draining is true if this endpoint should be drained
+	Draining bool `json:"draining" yaml:"draining"`
```

Comment:
can we add the `omitempty` tags here for `json` and `yaml`, it will eliminate the need to define this field in every test yaml